projects/sbb-esta/angular-public/src/lib/usermenu/_usermenu.scss

```diff
@@ -15,6 +15,11 @@ $borderActive5K: solid toPx(4 * $scalingFactor5k) $sbbColorGrey;
   width: 288px;
   height: 48px;
   line-height: 1.533;
+  border-top: 2px solid transparent;
+  border-left: 2px solid transparent;
+  border-right: 2px solid transparent;
+  border-bottom: 2px solid transparent;
+  outline: 0;
```

Comment:
Unless there is an alternative hightlight on focus, hiding outline is a problem for accessibility. So this should be removed.

Reply:
Yes you are totally right I will change to border: 2px solid transparent. 
And will also remove outline: 0.